test/e2e/util.go

```diff
@@ -346,7 +346,7 @@ func catalogSourceRegistryPodSynced(catalog *operatorsv1alpha1.CatalogSource) bo
 	if connState != nil {
 		state = connState.LastObservedState
 	}
-	fmt.Printf("waiting for catalog pod %v to be available (for sync) - %s\n", catalog.GetName(), state)
+	fmt.Printf("waiting for catalog pod %s/%s to be available (for sync) - %s\n", catalog.GetNamespace(), catalog.GetName(), state)
```

Comment:
Always fully qualify the object you're waiting on so that folks debugging a failing test know where to look.